vector/src/main/java/im/vector/services/EventStreamService.java

```diff
@@ -248,6 +248,13 @@ public void onBingEvent(Event event, RoomState roomState, BingRule bingRule) {
             //Log.d(LOG_TAG, "onBingEvent : the event " + event);
             //Log.d(LOG_TAG, "onBingEvent : the bingRule " + bingRule);
 
+            boolean mAllowVoip = getResources().getBoolean(R.bool.allow_voip);
```

Comment:
This change is useless.
The incoming call events are ignored in the sdk by the MXCallManager when the flavor "withoutVoip" is selected.
(see https://github.com/matrix-org/matrix-android-sdk/blob/develop/matrix-sdk/src/main/java/org/matrix/androidsdk/call/MXCallsManager.java#L382).

Please revert your changes in `EventStreamService.java`

Reply:
ok I agree we need to do something here
But I would prefer to check session.isVoipSupported() instead of this boolean allow_voip

Here is the suggested code:
````
if (event.isCallEvent()) {
                // Check whether the voip call is supported
                MXSession session = Matrix.getMXSession(getApplicationContext(), event.getMatrixId());
                if (null == session || !session.isVoipCallSupported()) {
                    Log.d(LOG_TAG, "ignore call event : voip not allowed");
                    return;
                }
            }
`````

Reply:
Done. And your are right, the allow_voip is useless now, I removed it.